src/java/Main.java

```diff
@@ -0,0 +1,4 @@
+package java;
+
+public class Main {
```

Comment:
I suppose you should rename class method

Reply:
I'm sure all OK ))